src/manifolds/PowerManifold.jl

```diff
@@ -404,154 +402,154 @@ function injectivity_radius(M::AbstractPowerManifold, x)
     for i in get_iterator(M)
         result += inner(
             M.manifold,
-            _read(M, rep_size, x, i),
-            _read(M, rep_size, v, i),
-            _read(M, rep_size, w, i),
+            _read(M, rep_size, p, i),
+            _read(M, rep_size, X, i),
+            _read(M, rep_size, Y, i),
         )
     end
     return result
 end
 
 is_default_metric(::AbstractPowerManifold, ::PowerMetric) = Val(true)
 
-function isapprox(M::AbstractPowerManifold, x, y; kwargs...)
+function isapprox(M::AbstractPowerManifold, p, q; kwargs...)
     result = true
     rep_size = representation_size(M.manifold)
     for i in get_iterator(M)
         result &= isapprox(
             M.manifold,
-            _read(M, rep_size, x, i),
-            _read(M, rep_size, y, i);
+            _read(M, rep_size, p, i),
+            _read(M, rep_size, q, i);
             kwargs...,
         )
     end
     return result
 end
-function isapprox(M::AbstractPowerManifold, x, v, w; kwargs...)
+function isapprox(M::AbstractPowerManifold, p, X, Y; kwargs...)
     result = true
     rep_size = representation_size(M.manifold)
     for i in get_iterator(M)
         result &= isapprox(
             M.manifold,
-            _read(M, rep_size, x, i),
-            _read(M, rep_size, v, i),
-            _read(M, rep_size, w, i);
+            _read(M, rep_size, p, i),
+            _read(M, rep_size, X, i),
+            _read(M, rep_size, Y, i);
             kwargs...,
         )
     end
     return result
 end
 
-@doc doc"""
-    log(M::AbstractPowerManifold, x, y)
+@doc raw"""
+    log(M::AbstractPowerManifold, p, q)
 
-Compute the logarithmic map from `x` to `y` on the [`AbstractPowerManifold`](@ref) `M`,
+Compute the logarithmic map from `p` to `q` on the [`AbstractPowerManifold`](@ref) `M`,
 which can be computed using the base manifolds logarithmic map elementwise.
 """
 log(::AbstractPowerManifold, ::Any...)
 
-function log!(M::AbstractPowerManifold, v, x, y)
+function log!(M::AbstractPowerManifold, X, p, q)
     rep_size = representation_size(M.manifold)
     for i in get_iterator(M)
         log!(
             M.manifold,
-            _write(M, rep_size, v, i),
-            _read(M, rep_size, x, i),
-            _read(M, rep_size, y, i),
+            _write(M, rep_size, X, i),
+            _read(M, rep_size, p, i),
+            _read(M, rep_size, q, i),
         )
     end
-    return v
+    return X
 end
 
 
-@doc doc"""
+@doc raw"""
     manifold_dimension(M::PowerManifold)
 
 Returns the manifold-dimension of an [`PowerManifold`](@ref) `M`
-$=\mathcal N = (\mathcal M)^{n_1,\ldots,n_d}, i.e. with $n=(n_1,\ldots,n_d)$ the array
-size of the power manifold and $d_{\mathcal M}$ the dimension of the base manifold
-$\mathcal M$, the manifold is of dimension
+$=𝒩 = (ℳ)^{n_1,…,n_d}, i.e. with $n=(n_1,…,n_d)$ the array
+size of the power manifold and $d_{ℳ}$ the dimension of the base manifold
+$ℳ$, the manifold is of dimension
```

Comment:
use `\dim` below

Reply:
Ah yes, we should use `\dim` more consistently.